packages/manager/modules/models/billing/DebtAccount.class.js

```diff
@@ -0,0 +1,19 @@
+import get from 'lodash/get';
+
+export default class DebtAccount {
```

Comment:
Could you please add documentation everywhere ? We could use it to make a glossary

Reply:
The name and payload reflects the API and looks quite explicit but sure if it can help the understanding

Reply:
A glossary is not only for the devs :)

Reply:
I agree but I'm not sure it is our scope to define the billing glossary